examples/interactive_play.py

```diff
@@ -72,7 +72,7 @@
 except ImportError:
     pygame = None
 
-DEFAULT_CFG = "benchmark/rearrange/play.yaml"
+DEFAULT_CFG = "benchmark/rearrange/play_stretch.yaml"
```

Comment:
Are we migrating our default to Stretch at this time? Otherwise this should be a comment option or terminal arg. @ASzot

Reply:
Thanks for this! I fixed this.